tests/test_eigendistortion.py

```diff
@@ -193,16 +192,20 @@ def test_jac_vec_prod(self, state):
     def test_fisher_vec_prod(self, state):
         x, y, x_dim, y_dim, k = state
 
-        V = torch.randn((x_dim, k), device=DEVICE)
+        V, _ = torch.linalg.qr(torch.ones((x_dim, k), device=DEVICE), "reduced")
+        U = V.clone()
```

Comment:
- same comment about `torch.linalg` and torch 1.10
- why use the cloned version?

Reply:
- torch.linalg intro'd in 1.8.
- cloned bc i'm checking against values and didn't want to accidentally refer to same underlying object